source/style-guide/style/language-selectors.txt

```diff
@@ -0,0 +1,108 @@
+.. _language-selectors:
+
+=============================
+Method and Language Selectors
+=============================
+
+.. default-domain:: mongodb
+
+Method and language selectors let you provide dynamic content based on 
+the selected method and language.
+
+Method and Language Selector Anatomy
+------------------------------------
+
+A method and language selector contains the following parts:
+
+1. Radio boxes with different method options (up to six)
+2. Optional method description
+3. Language selector drop-down (drivers only)
+
+.. figure:: /images/language-selectors-anatomy.png
+   :alt: method and language selectors anatomy
+   :align: center
+   :figwidth: 700px
+
+Method and Language Selector Syntax
+-----------------------------------
+
+The following example shows the syntax for a method selector:
+
+.. code-block::
+
+   .. method-selector::
+
+      .. method-option::
+         :id: driver
+
+         .. method-description::
+
+            .. include:: /includes/description-drivers.rst
+      
+            .. tabs-selector:: drivers
+
+The following example shows the syntax for a language selector:
+
+.. code-block::
+
+   .. tabs-drivers::
+         
+      .. tab:: 
+         :tabid: compass
+         
+         .. include:: /includes/steps-gswa-connect-to-cluster-compass.rst
+         
+      .. tab::
+         :tabid: csharp
+         
+         .. include:: /includes/steps-gswa-driver-connect-csharp.rst
+         
+      .. tab:: 
+         :tabid: go
+         
+         .. include:: /includes/steps-gswa-driver-connect-go.rst
+         
+      .. tab:: 
+         :tabid: java-sync
+         
+         .. include:: /includes/steps-gswa-driver-connect-java.rst
+         
+      .. tab:: 
+         :tabid: python
+                  
+          .. include:: /includes/steps-gswa-driver-connect.rst
+         
+      .. tab::
+         :tabid: nodejs
+               
+         .. include:: /includes/steps-gswa-driver-connect-node.rst
+
+Method and Language Selector Guidelines
+---------------------------------------
+
+When you create method and language selectors, use the following 
+guidelines:
+
+- Add a language selector only when :guilabel:`Driver` is the selected 
+  method.
+
+  .. figure:: /images/language-selector-driver-method.png
```

Comment:
[q] I'm confused by this diagram, should a different method be highlighted in the ❌ example? Both have **Driver** selected.